packages/storyshots/stories/__test__/storyshots.test.js

```diff
@@ -1,2 +0,0 @@
-import initStoryshots from '../../src';
```

Comment:
yes temporary indeed, the implementation was a bad smoke-test. Someone needs to do a thorough analysis on how to test it.